src/nvim/api/vim.c

```diff
@@ -1328,7 +1328,7 @@ Boolean nvim_paste(String data, Boolean crlf, Integer phase, Error *err)
   if (!cancel && !(State & CMDLINE)) {  // Dot-repeat.
     for (size_t i = 0; i < lines.size; i++) {
       String s = lines.items[i].data.string;
-      assert(data.size <= INT_MAX);
+      assert(s.size <= INT_MAX);
```

Comment:
that doesn't quite look like (just) a typo to me...

Reply:
https://github.com/neovim/neovim/pull/17752

Reply:
Yes, I saw that, but that doesn't change my point: functional code changes -- even if minor -- are not "typos" and shouldn't be swept under that rug.

Reply:
Yes. I'll remove it.

Reply:
(Not sure the original PR can be reopened, though.)

Reply:
OK, will create a new one.

Reply:
Never mind, he reopened his PR.